documentation/documentation.gradle

```diff
@@ -77,7 +77,7 @@ asciidoctor {
 				'mainDir': project.sourceSets.main.java.srcDirs[0],
 				'testDir': project.sourceSets.test.java.srcDirs[0],
 				'testResourcesDir': project.sourceSets.test.resources.srcDirs[0],
-				'source-highlighter': 'coderay', // TODO switch to 'rouge' once supported by the html5 backend
+				'source-highlighter': 'rouge',
```

Comment:
See Dan's comments in the _related_ [Spring JIRA issue](https://jira.spring.io/browse/SPR-14997?focusedCommentId=136524&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-136524). :wink: